src/axom/inlet/examples/document_generation.cpp

```diff
@@ -0,0 +1,146 @@
+// usage : ./document_generation_example --enableDocs --deck lua_file 
+
+#include "axom/inlet/SphinxDocWriter.hpp"
+#include "axom/inlet/Inlet.hpp"

```

Comment:
Change these two includes to:

#include "axom/inlet.hpp"

We have a unified header for each component that we want users to use.  There is no way for you to know this =)